NickvisionMoney.Shared/Models/Account.cs

```diff
@@ -308,7 +308,7 @@ public bool Login(string? password)
             //Set Password
             if (IsEncrypted)
             {
-                if (string.IsNullOrEmpty(password))
+                if (string.IsNullOrWhiteSpace(password))
```

Comment:
Didn't catch this earlier. This one would prevent accessing accounts stored with an older version which has its password set as a whitespace. I don't why would anyone do that but since we allowed it earlier, someone out there could have done it

Reply:
Yes I'd revert this back to `string.IsNullOrEmpty()` for backward compatibility

Reply:
> While it is possible, it is quite unlikely. Lets leave it like this and if somebody complains, we can restore it.

Imo, that is not a good approach. Many people, even though on linux don't know how to create an issue 😅. I agree that is a very rare use case though.

Reply:
When though it is a very rare case, I believe we shouldn't throw those users away and break it for them. `string.IsNullOrEmpty` it should stay...

As mentioned below, we want to keep whitespace passwords anyway so

Reply:
reverted